assets/dev/js/editor/editor-base.js

```diff
@@ -1255,9 +1255,11 @@ export default class EditorBase extends Marionette.Application {
 			devices = this.breakpoints.getActiveBreakpointsList( { largeToSmall: true, withDesktop: true } ),
 			newControlsStack = {};
 
+		// Set the desktop to be the fist device, so desktop will the the parent of all devices.
+		devices.unshift( 'desktop' );
```

Comment:
Are you sure this is not creating 2 desktop items in the array?

Reply:
> Are you sure this is not creating 2 desktop items in the array?

Pretty sure. I removed the 'widthDesktop' property in the devices retrieval few lines above.

Reply:
> Are you sure this is not creating 2 desktop items in the array?

Updated the commit, notice the `withDesktop` property above.